CHANGES.rst

```diff
@@ -1,3 +1,8 @@
+0.1.13 (2024-01-14)
```

Comment:
It's great to add your changes to the changelog, but please allow maintainers to manage the release versions.

This can be changed to:

```suggestion
development (unreleased)
```